_overviews/compiler-options/optimizer.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: singlepage-overview
+title: Optimizer
+---
+
+**[Lukas Rytz](https://github.com/lrytz) (November 7, 2018)**
```

Comment:
s/November 7, 2018/2018/

Reply:
OK, but it was fun knowing what Lukas was doing as US election results rolled in. That would be midterm results in the Trump admin. Something about the drift in the political landscape made Lukas go, I'd better right the ship of state with a blog about the optimizer. And that's why we are where we are today, modulo Ukraine.